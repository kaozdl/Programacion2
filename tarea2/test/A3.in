insertar_lista (20,conejo) (10,mono) (30,mono) (15,caballo) .
modificar_lista 10 30
modificar_lista 15 20
Fin
