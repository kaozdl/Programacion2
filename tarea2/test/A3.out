1>(0,primero)(20,conejo)(10,mono)(30,mono)(15,caballo)
2>(0,primero)(20,conejo)(30,mono)(10,mono)(15,caballo)
(0,primero)(10,mono)(20,conejo)(30,mono)(15,caballo)
3>(0,primero)(10,mono)(15,caballo)(30,mono)(20,conejo)
(0,primero)(10,mono)(15,caballo)(30,mono)(20,conejo)
4>Fin.
